extension/content.js

```diff
@@ -60,6 +62,25 @@ function addToggleBtn() {
 	}
 }
 
+function fixupOtherButtons() {
+	const sidebar = document.querySelector('.file-navigation .right');
+	if (!sidebar) {
+		return;
+	}
+
+	const buttons = Array.from(sidebar.getElementsByClassName('btn btn-sm'));
+	buttons.forEach(button => {
```

Comment:
`querySelector` is well suited to this use case, as it returns the first matching instance of the selector. This is great for finding one-off portions of the DOM like the sidebar of buttons but not great for finding a particular button in that sidebar.

While using `getElementsByClassName` on the whole page would probably be expensive, getting the buttons from the sidebar is an inexpensive call. I ran it the code in the profiler and `fixupOtherButtons` consistently ran in sub-millisecond time.